nbs/13a_learner.ipynb

```diff
@@ -103,10 +103,10 @@
     "    valid_ds = get_data(n_valid)\n",
     "    device = default_device() if cuda else None\n",
     "    if tfmdDL:\n",
-    "        train_dl = TfmdDL(train_ds, bs=bs, shuffle=True, num_workers=0)\n",
+    "        train_dl = TfmdDL(train_ds, bs=bs, shuffle=True, num_workers=0, drop_last=True)\n",
```

Comment:
added `drop_last=True` to the synth_dbunch to more accurately represent a real dls.  Would it be better to make this an argument that defaults to True instead?